src/Form/PostType.php

```diff
@@ -16,22 +13,19 @@ class PostType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('category', ChoiceType::class, [
-                'choices' => Post::CATEGORY_CHOICES,
-                'choice_attr' => function ($choice, $key, $value) {
-                    return ['class' => 'post_category_'.$value];
-                },
+            ->add('title', TextType::class, [
+                'required' => true,
+            ])
+            ->add('content', TextareaType::class, [
+                'required' => true,
             ])
-            ->add('title', TextType::class)
-            ->add('content', TextareaType::class)
-            ->add('submit', SubmitType::class)
         ;
     }
 
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'data_class' => Post::class,
```

Comment:
Why do you remove this line?

Reply:
this class is only used to reduce code duplication for forms as explained here https://symfony.com/doc/current/form/inherit_data_option.html